srcpkgs/xmoto/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'xmoto'
 pkgname=xmoto
 version=0.6.1
-revision=2
+revision=3
 build_style=cmake
 build_helper="qemu"
-configure_args="-DOpenGL_GL_PREFERENCE=GLVND -DXMOTO_PACK=manual"
+configure_args="-DCMAKE_BUILD_TYPE=Release -DOpenGL_GL_PREFERENCE=GLVND -DXMOTO_PACK=manual"
```

Comment:
Is there a reason for this?

Reply:
It appears on the instructions to build (upstream) I am assuming it removes debugs.

Reply:
the build style sets CMAKE_BUILD_TYPE=None, which should be fine usually, unless they do some weird special handling

Reply:
Reverted